knr1050Sup/knr1050.db

```diff
@@ -656,6 +670,20 @@ record(bi, "$(P)ERR_IN") {
     info(INTEREST, "HIGH")
 }
 
+record(stringin, "$(P)ERROR:STR") {
+    field(DESC, "Input error return")
+	field(DTYP, "stream")
+	field(SCAN, "I/O Intr")
```

Comment:
Please use spaces instead of tabs for whitespace

Reply:
Converted tabs to spaces.